xontrib-cookiecutter v2 (#2)
xonsh/xontrib-cookiecutter: README.md

<p align="center">
A <a href="https://github.com/audreyr/cookiecutter">cookiecutter</a> template for <a href="https://github.com/xonsh/xonsh">xonsh</a> contributions called <a href="https://xon.sh/xontribs.html">xontribs</a>.
</p>

<p align="center">  
If you like the template click ⭐ on the repo.
</p>

## Why use this template?

This template includes good pack of prebuilt files: 

* xontrib promotion instructions in the README 
* `setup.py` file to make and install PyPi package easily 
* `.gitattributes` file to enable Github syntax highlighting for `*.xsh` files
* `.gitignore` file with standard list of directories to ignore
* `MANIFEST.in` file to make Conda feedstock easily
* `LICENSE` file with standard MIT license
* `requirements.txt` file to make reference to xonsh on Github and promote xontrib

## Create new xontrib

> To create xontrib from the template just run:

Reviewer: ```suggestion
To create a `xontrib` from this template just run:
```

If you like the template click ⭐ on the repo.
</p>

## Why use this template?

This template includes good pack of prebuilt files: 

* xontrib promotion instructions in the README 
* `setup.cfg` file to make and install PyPi package easily 
* `.gitattributes` file to enables Github syntax highlighting for `*.xsh` files

> * `.gitignore` file with standart list of directories to ignore

Reviewer 2: ```suggestion
* `.gitignore` file with standard list of directories to ignore
```
